lblod/frontend-organization-portal
# fix bug chrome & aumodal

OP-1469

How to reproduce the bug (without the fix):
- make sure you're using Chrome 102
- open the contact details modal (edit a position)
- press either cancel or selecteer button
- the page is blocked & you can't do anything

Happens to all environments (dev, qa)

this is temporary, once the issue is fixed in appuniversum, we should do OP1474
